hack/local-up-cluster.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright 2014 The Kubernetes Authors All rights reserved.
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
These should only be 2015 in files that you added just now-- I'm pretty sure this file isn't one of those. :)